src/egl/drivers/dri2/platform_android.c

```diff
@@ -66,6 +66,7 @@ enum {
 static const struct droid_yuv_format droid_yuv_formats[] = {
    /* Native format, YCrCb, Chroma step, DRI image FourCC */
    { HAL_PIXEL_FORMAT_YCbCr_420_888,   0, 2, __DRI_IMAGE_FOURCC_NV12 },
+   { HAL_PIXEL_FORMAT_P010_INTEL,      0, 4, __DRI_IMAGE_FOURCC_P010 },
```

Comment:
Lets start getting rid of all this custom formats by replacing it with HAL_IMPLEMENTATION_DEFINED